web/cypress.json

```diff
@@ -2,5 +2,7 @@
   "viewportWidth": 1920,
   "viewportHeight": 1080,
   "baseUrl": "http://localhost:8000",
-  "video": true
+  "video": true,
+  "numTestsKeptInMemory": 0,
```

Comment:
any resources about this field?

Reply:
see: https://docs.cypress.io/guides/references/configuration#Options